sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/BlobCheckpointStore/BlobsCheckpointStoreInternalTests.cs

```diff
@@ -1202,6 +1207,11 @@ public async Task GetCheckpointPreferredNewCheckpointOverLegacy()
             Assert.That(checkpoint, Is.Not.Null, "A single checkpoint should have been returned.");
             Assert.That(checkpoint.StartingPosition, Is.EqualTo(EventPosition.FromOffset(14, false)));
             Assert.That(checkpoint.PartitionId, Is.EqualTo("0"));
+
+            Assert.That(checkpoint, Is.InstanceOf<BlobCheckpointStoreInternal.BlobStorageCheckpoint>());
+            var blobCheckpoint = (BlobCheckpointStoreInternal.BlobStorageCheckpoint)checkpoint;
+            Assert.AreEqual(14, blobCheckpoint.Offset);
```

Comment:
Could we please stick to the `Assert.That` format and include a failure message since there are multiple asserts in this test?